### ISSUE TITLE
Youtube-dl recursively downloading youtube channels.

### ISSUE THREAD

**Issue by Kelgot** [ytdl-org/youtube-dl#27099]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x ] I've verified that I'm running youtube-dl version **2020.11.19**
- [x ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x ] I've searched the bugtracker for similar bug reports including closed ones
- [ x] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v']
[debug] Encodings: locale cp65001, fs mbcs, out cp65001, pref cp65001
[debug] youtube-dl version 2020.11.19
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.18362
[debug] exe versions: ffmpeg git-2020-05-27-8b5ffae, ffprobe git-2020-05-27-8b5ffae
[debug] Proxy map: {}
```

## Description

Youtube-dl is recursively searching youtube channels for more and more videos to download instead of downloading videos strictly from the user channel.

EXAMPLE: 
Channel url: https://www.youtube.com/channel/UCeWLUT2SaZ2rMVYkuLxZlJw
Youtubedl command: 

youtube-dl -i --download-archive N:\Software\youtube-dl\archive.txt -w --write-description --write-info-json --write-thumbnail --all-subs --add-metadata -f bestvideo+bestaudio --merge-output-format mkv "https://www.youtube.com/channel/UCeWLUT2SaZ2rMVYkuLxZlJw" -o G:\Archiving\YouTube\%(channel_id)s\%(upload_date)s_%(id)s\%(upload_date)s.%(id)s.%(ext)s

The channel's videos will be downloaded at the start. Once finished, it starts to download all of the playlist this channel has created. Then, it will go to the "Channels" tab and download all of these channels. Then, it will download the playlists of these channels and the related channels to these channels. This recursion does not end until the chain is broken by someone not having any playlists or related channels. This did not happen in the October release that I had. I will upload a text file of output that shows the output when running the above command. 

[log.txt](https://github.com/ytdl-org/youtube-dl/files/5570044/log.txt)

**Comment by remitamine** [comment]
> Youtube-dl is recursively searching youtube channels for more and more videos to download instead of downloading videos strictly from the user channel.

there is a specific tab URL(Videos tab) for this(https://www.youtube.com/c/Vandun/videos).

**Comment by Lightsockie** [comment]
Why the change in functionality :^(

**Comment by remitamine** [comment]
website changes.

**Comment by makeworld-the-better-one** [comment]
This has been fixed in version 2020.12.02. The commit of the fix is 132aece1ed824aafd849ef63dcb44c835eab6ee9.
